src/include/execution/sql/thread_state_container.h

```diff
@@ -4,15 +4,10 @@
 #include <memory>
 #include <vector>
 
-#include "common/strong_typedef.h"
 #include "execution/sql/memory_pool.h"
 
-namespace terrier::execution::exec {
-class ExecutionSettings;
-}  // namespace terrier::execution::exec
-
 namespace terrier::execution::sql {
-
+using byte = std::byte;
```

Comment:
Ugh, I didn't realize `byte` is defined in `strong_typedef.h`. Re-include that header file.